tools/dependency/release_dates.py

```diff
@@ -51,6 +60,49 @@ def verify_and_print_latest_release(dep, repo, metadata_version, release_date):
         print(
             f'{Fore.YELLOW}*WARNING* {dep} has a newer release than {metadata_version}@<{release_date}>: '
             f'{latest_release.tag_name}@<{latest_release.created_at}>{Style.RESET_ALL}')
+        # check for --cron flag, To run this only on github action schedule
+        # and it does not bloat CI on every push
+        if create_issue:
+            create_issues(dep, repo, metadata_version, release_date, latest_release)
+
+
+# create issue for stale dependency
+def create_issues(dep, repo, metadata_version, release_date, latest_release):
+    """Create issues in GitHub.
+
+    Args:
+        dep : name of the deps
+        repo : BaseURL of the repo
+        metadata_version :
+        release_date : old release_date
+        latest_release : latest_release (name and date )
+    """
+    access_token = os.getenv('GITHUB_TOKEN')
+    git = github.Github(access_token)
+    repo = git.get_repo('envoyproxy/envoy')
+
+    # Find GitHub label objects for LABELS.
+    labels = []
+    for label in repo.get_labels():
```

Comment:
I was using this `for` loop code from [create_issue](https://github.com/envoyproxy/envoy/blob/a12869fa9e9add4301a700978d5489e6a0cc0526/tools/deprecate_version/deprecate_version.py#L53) ..  

but i think its no use because now as LABELS on line 27 can be changed anytime and these issue Labels will be a part of this envoy repo. 

Should i remove these lines and use LABELS explicitly in `repo.create_issues`

Reply:
ping @asraa

Reply:
the `get_labels()` is just making sure that the LABELS in line 27 are actual repo issue labels. Why not? It should be that all of the LABELS are actually in get_labels (see line 89 for the check).

Reply:
ok i was confirming. As LABELS are hard coded and will already be present in the repo labels.